frontend/app/component/pollingstation/PollingStationFormNavigation.tsx

```diff
@@ -28,6 +30,11 @@ export function PollingStationFormNavigation({ pollingStationId, election }: Pol
   //one time flag to prioritize user navigation over controller navigation
   const overrideControllerNavigation = React.useRef<string | null>(null);
 
+  const isPartOfInputFlow = React.useCallback(
```

Comment:
Should be named `isPartOfDataEntryFlow` now that #345 is merged.

Reply:
Done!

Reply:
If this is done, shouldn't I see `isPartOfDataEntryFlow` here instead of `isPartOfInputFlow`?

Reply:
I had merged main into this branch, but forgot to change this const, done in a63ac6daa40540d684fea63993ece93a92f4b388 now